custom/intrahealth/sqldata.py

```diff
@@ -1684,7 +1684,7 @@ class RecoveryRateByPPSData(VisiteDeLOperateurDataSource):
     show_total = True
     custom_total_calculate = True
 
-    def calculate_total_row(self, data):
+    def calculate_total_row(self, data, tmp):
```

Comment:
It's not very clear what `tmp` means or why you need it here instead of deriving it from `data` or a database query

Reply:
I am going as well to clean up the naming - this one is not very useful.